tests/tests/test_base58.py

```diff
@@ -0,0 +1,46 @@
+# Tests adapted from base58:
+# https://github.com/keis/base58/blob/master/test_base58.py
+
+from libs.common.bitcoin.base58 import (
+    encode, decode, encode_check, decode_check)
```

Comment:
If we tweak `sys.path` like in my suggestion we can change it to
```suggestion
from bitcoin.base58 import encode, decode, encode_check, decode_check
```

Reply:
Fixed